radio/src/gui/horus/bitmapbuffer.h

```diff
@@ -138,7 +140,15 @@ class BitmapBuffer: public BitmapBufferBase<uint16_t>
 
     void drawBitmapPatternPie(coord_t x0, coord_t y0, const uint8_t * img, LcdFlags flags, int startAngle, int endAngle);
 
-    static BitmapBuffer * load(const char * filename);
+    static BitmapBuffer * load_bmp(const char * filename);
+    static BitmapBuffer * load_stb(const char * filename);
+    static BitmapBuffer * load(const char * filename) {
+#if defined(USE_STB)
```

Comment:
We could completely remove my code, no? There is not so much interest in keeping both systems!